File: user-management-service/src/main/java/com/user_management_service/config/SwaggerConfig.java
package com.user_management_service.config;

import org.springdoc.core.models.GroupedOpenApi;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class SwaggerConfig {

    @Bean
    public GroupedOpenApi customOpenApi(){
        return GroupedOpenApi.builder()
                .group("user-management-api")
                .packagesToScan("com.user_management_service.controller")
                .build();
    }

}

File: course-management-service/src/main/kotlin/com/course_management_service/config/SwaggerConfig.kt
package com.course_management_service.config

import org.springdoc.core.models.GroupedOpenApi
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class SwaggerConfig {
    @Bean
    fun customOpenApi(): GroupedOpenApi {
        return GroupedOpenApi.builder()
            .group("user-management-api")
            .packagesToScan("com.course_management_service")
            .build()
    }
}